### ISSUE TITLE
token missing in signIn callback

### ISSUE THREAD

**Issue by smarwei** [StudioAdriatic/PGSGP#6]
Hi,
I was hoping to use this plugin to authenticate with my backend. 
The documentation says that a "User token for backend use" is returned in the sign in callback. However, this attribute is missing for me.

**Comment by kristijandraca** [comment]
> authenticate with my backend.

Do you mean auth with Google Play Game Services?

Also, can you please run 
`adb logcat -s godot`

and paste logs here?

**Comment by smarwei** [comment]
> Do you mean auth with Google Play Game Services?
Users can create an account on my server, but I want them to be able to log in with their Google account as well. I thought Google Play Services was the way to go.

the adb logcat output is as follows:
$ ./adb logcat -s godot
--------- beginning of crash
--------- beginning of system
--------- beginning of main
01-05 21:48:47.841 15464 15501 I godot   : Godot Engine v3.4.2.stable.official.45eaa2daf - https://godotengine.org
01-05 21:48:47.849 15464 15501 I godot   : OpenGL ES 3.0 Renderer: Adreno (TM) 618
01-05 21:48:47.853 15464 15501 I godot   : OpenGL ES Batching: ON
01-05 21:48:47.871 15464 15501 I godot   :
01-05 21:48:47.927 15464 15501 E godot   : ERROR: No library set for this platform
01-05 21:48:47.927 15464 15501 E godot   :    at: initialize (modules/gdnative/gdnative.cpp:293) - No library set for this platform
01-05 21:48:48.476 15464 15464 I godot   : Sign in data is valid
01-05 21:48:48.476 15464 15464 I godot   : Using cached signin data
01-05 21:48:48.735 15464 15464 I godot   : Sign in data is valid
01-05 21:48:49.123 15464 15501 I godot   : {banner_image_landscape_url:https://lh3.googleusercontent.com/AogCtkMsMmscGAqvG5xC5h5Oy0XwlL9nIE9fq8HR2rXlRl5cInsZxnhrkqH0zlPE0AmGoe_x0WzzSck5Tb0=s2210, banner_image_portrait_url:https://lh3.googleusercontent.com/stE_SLQUnI6lquMdEO8FYQrtjCHHYa_41MjBFzTwEH0A8X2j4qBtcXIwcaXr9lKszPl6wzShqNE0fnsHvA=s2210, display_name:UnsoldSuperstition42, hi_res_image_url:https://lh3.googleusercontent.com/natAaTJqH8o8n952ui0MAyA6SfZCN5izU4l9iLvC7pvaE6ysFmDepyq7RP7OyPm8Fnk=s517-ns, icon_image_url:https://lh3.googleusercontent.com/natAaTJqH8o8n952ui0MAyA6SfZCN5izU4l9iLvC7pvaE6ysFmDepyq7RP7OyPm8Fnk=s220-ns, level_info:{current_level:{level_number:8, max_xp:50000, min_xp:40000}, current_xp_total:47000, last_level_up_timestamp:1509888760090, next_level:{level_number:9, max_xp:60000, min_xp:50000}}, name:Ame Weiss, player_id:g10935440675213820142, title:General}
01-05 21:48:49.123 15464 15501 I godot   : {banner_image_landscape_url:https://lh3.googleusercontent.com/AogCtkMsMmscGAqvG5xC5h5Oy0XwlL9nIE9fq8HR2rXlRl5cInsZxnhrkqH0zlPE0AmGoe_x0WzzSck5Tb0=s2210, banner_image_portrait_url:https://lh3.googleusercontent.com/stE_SLQUnI6lquMdEO8FYQrtjCHHYa_41MjBFzTwEH0A8X2j4qBtcXIwcaXr9lKszPl6wzShqNE0fnsHvA=s2210, display_name:UnsoldSuperstition42, hi_res_image_url:https://lh3.googleusercontent.com/natAaTJqH8o8n952ui0MAyA6SfZCN5izU4l9iLvC7pvaE6ysFmDepyq7RP7OyPm8Fnk=s517-ns, icon_image_url:https://lh3.googleusercontent.com/natAaTJqH8o8n952ui0MAyA6SfZCN5izU4l9iLvC7pvaE6ysFmDepyq7RP7OyPm8Fnk=s220-ns, level_info:{current_level:{level_number:8, max_xp:50000, min_xp:40000}, current_xp_total:47000, last_level_up_timestamp:1509888760090, next_level:{level_number:9, max_xp:60000, min_xp:50000}}, name:Ame Weiss, player_id:g10935440675213820142, title:General}

**Comment by smarwei** [comment]
I have finally solved the problem.
The root cause was that I used the Client-ID of my Android application as the request_token. In fact, I had to create additional credentials for a web application and use its credentials instead. (see https://github.com/cgisca/PGSGP/pull/48)
It took me some time to come to realize that this actually works, because the initial token passed when starting the App the first time is apparently used until the application is uninstalled. So it is not enough to reinstall the application in Godot, it has to be manually uninstalled beforehand if a wrong token has already been used.
